js/doka/element-addeventlistener/demos/keyboard-events.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Обработка событий клавиатуры — Element.addEventListener() — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" >
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
+  <style>
+    * {
+      box-sizing: border-box;
+      margin: 0;
+    }
+
+    body {
+      padding: 5%;
+      background-color: #18191c;
+      color: #ffffff;
+      font-family: "Roboto", sans-serif;
+      font-size: 18px;
+      text-align: center;
+      overflow: hidden;
+    }
+
+    .container {
+      display: inline-block;
+      text-align: left;
+    }
+
+    h2 {
+      font-size: 24px;
+      font-weight: 500;
+    }
+
+    label {
+      display: block;
+    }
+
+    label,
+    input,
+    #event,
+    #result {
+      margin-top: 20px;
+    }
+
+    input {
+      display: block;
+      width: 100%;
+      border: 1px solid #ffffff;
+      margin-top: 10px;
+      padding: 15px;
+      background-color: #18191c;
+      color: #ffffff;
+      font-family: "Roboto", sans-serif;
+      font-size: 18px;
+      font-weight: 300;
+    }
+
+    input::placeholder {
+      color: rgba(255, 255, 255, 0.6);
+      font-size: 18px;
+    }
+
+    input:focus {
+      border-color: #1a5ad7;
+      outline: none;
+    }
+
+    #result {
+      color: #1A5AD7;
+    }
+
+    code {
+      font-family: "Roboto Mono", monospace;
+      font-size: 0.9em;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <h2>Пример работы с событиями ввода с клавиатуры</h2>
+    <label for="field">
+      Начинайте вводить сообщение, после чего нажмите <code>Enter</code>
+    </label>
+    <input id="field" type="text" placeholder="Введите сообщение">
+    <div id="event">Ожидание ввода...</div>
+    <div id="result"></div>
+  </div>
+
+<script>
+  function handleKey(event) {
+    let msg = 'Нажата клавиша: <code>' + event.key + '</code>';
```

Comment:
Давай использоваать `const` везде, где устанавливаем значние переменной один раз? 

```suggestion
    const msg = 'Нажата клавиша: <code>' + event.key + '</code>';
```